toolchain/check/keyword_modifier_set.h

```diff
@@ -53,7 +55,7 @@ class KeywordModifierSet {
     Method = Abstract | Impl | Virtual,
     ImplDecl = Extend | Final,
     Interface = Default | Final,
-    Decl = Class | Method | ImplDecl | Interface | Export | Returned,
+    Decl = Class | Method | Impl | Interface | Export | Returned,
```

Comment:
The change from `ImplDecl` -> `Impl` here is surprising to me. `Impl` is already a part of `Method` and refers to using `impl` as a modifier keyword on virtual methods (to be replaced by `override`, see #5253 ), as opposed to `ImplDecl` which is the modifiers allowed on an `impl` declaration. It would make more sense to delete this than use `Impl` here.

Reply:
Created https://github.com/carbon-language/carbon-lang/pull/5425 to make this change.